src/Controller/SiteController.php

```diff
@@ -33,6 +52,55 @@ public function index(): ResponseInterface
         return $response;
     }
 
+    public function contact(RequestInterface $request): ResponseInterface
+    {
+        $parameters = [];
+        if ($request->getMethod() === 'POST') {
+            $config = array_merge(
+                require $this->aliases->get('@root/config/params.php'),
+                require $this->aliases->get('@root/config/params-local.php'),
+            );
```

Comment:
Is there a way to get these from composer-config-plugin instead?

params-local.php is meant to be optional during `composer du` but here it would break.. using the composer-config-plugin compiled output seems like a better idea, but I'm not really familiar with its API.

Reply:
Yes:

```php
$config = require Builder::path('params');
```

It should load full config from `vendor/hiqdev/composer-config-plugin-output/params.php`.